src/plots/piePlot.ts

```diff
@@ -249,20 +246,31 @@ export module Plots {
       let sectorValueAccessor = Plot._scaledAccessor(this.sectorValue());
       let ds = this.datasets()[0];
       let data = dataToDraw.get(ds);
-      let filteredData = data.filter((d, i) => Plottable.Utils.Math.isValidNumber(sectorValueAccessor(d, i, ds)));
+      let filteredData = data.filter((d, i) => Pie._isValidData(sectorValueAccessor(d, i, ds)));
       dataToDraw.set(ds, filteredData);
       return dataToDraw;
     }
 
+    protected static _isValidData(value: any) {
+      return Plottable.Utils.Math.isValidNumber(value) && value >= 0;
+    }
+
     protected _pixelPoint(datum: any, index: number, dataset: Dataset) {
+      let scaledValueAccessor = Plot._scaledAccessor(this.sectorValue());
+      if (!Pie._isValidData(scaledValueAccessor(datum, index, dataset))) {
+        return { x: NaN, y: NaN};
```

Comment:
This isn't immediately relevant because `_pixelPoint` isn't invoked for invalid slices, but I wonder what we should be returning in these cases. A few options I see:
- A valid point, with invalid `x` and `y` properties
  - `NaN`s? `null`s? `undefined`s?
- `null` instead of a `Point`

The second option is more blatant, but runs the risk of blowing up downstream code if they aren't expecting invalid values.